### ISSUE TITLE
"It's now safe to turn off your computer" screen should be displayed after shutdown.

### ISSUE THREAD

**Issue by IdmFoundInHim** [felixrieseberg/windows95#112]
Shutdown does not seem to do much of anything. I just get a black screen with one white underscore:
![black screen with underscore](https://user-images.githubusercontent.com/30935172/50605137-650d6680-0e8f-11e9-9a62-1f017ce66851.png)

If shutdown does what it did on Windows 95 -- stop all running processes and wait for you to pull the plug -- there should be some assurance once this is finished. Preferably, the traditional message from Windows 95...
![windows 95 shutdown screen](https://i.imgur.com/keBwkpK.jpg)

**Comment by yzhang37** [comment]
Actually, this machine cannot recover from either Shutdown or Restart. If you make something changes to the hard disk and then choose **Start** | **Shutdown** | **Restart Computer**, scarcely can you recover it from the previous state.

**Comment by dxgldotorg** [comment]
It doesn't show the boot screen either if you discard the state.  Apparently it doesn't support the particular Mode-X mode used by the boot and shutdown screens.

**Comment by beedell-roke** [comment]
Sadly, this means I can't use 32-bit colour :(

**Comment by BaRRaKudaRain** [comment]
Is it very needed?

-----Original Message-----

**Comment by IdmFoundInHim** [comment]
Well, the *bug* part of this issue is fixing the shutdown. Right now, shutting down breaks the app. From: Ilya LapaSent: Sunday, May 12, 2019 5:48 PMTo: felixrieseberg/windows95Cc: IdmFoundInHim; AuthorSubject: Re: [felixrieseberg/windows95] "It's now safe to turn off your computer" screen should be displayed after shutdown. (#112) Is it very needed?-----Original Message-----From: "Roke Beedell" <notifications@github.com>Sent: ‎5/‎12/‎2019 4:13 PMTo: "felixrieseberg/windows95" <windows95@noreply.github.com>Cc: "Subscribed" <subscribed@noreply.github.com>Subject: Re: [felixrieseberg/windows95] "It's now safe to turn off yourcomputer" screen should be displayed after shutdown. (#112)Sadly this means I can't use 32-bit colour :(—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or mute the thread. —You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by beedell-roke** [comment]
> Is it very needed?

@BaRRaKudaRain, I suppose not, but by that reasoning nothing is.

What I'm saying is merely that it'd be a nice feature for certain games, for instance. However, as you said, it's not required, at all.

**Comment by ITAC85v2** [comment]
See my comment on issue #174

**Comment by felixrieseberg** [comment]
A lot has changed since this was filed: v5/v6 reworked restart, shutdown, and state persistence. Restart from the Start menu works now, your machine state is saved automatically when you quit, and it survives app updates. If a machine ever gets stuck, Settings → Machine State lets you reset the state or boot from scratch.

Closing this one out — if you still see trouble on the current release, please open a new issue.

**Comment by beedell-roke** [comment]
@felixrieseberg, how does any of that relate to this issue?
